NMDSdevopsServiceAdm/DataEngineering
# Validation add check of distinct values

# Description
Add in check of distinct values in categorical columns and tests for the check
Add in rules for merged cqc data

# How to test
unit tests passing
Run in branch here: https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:344210435447:execution:validation-add-check-of-distin-Ind-CQC-Filled-Post-Estimates-Pipeline:ac0788ec-c426-450b-8a57-7c1c753f115b
